changelog.json

```diff
@@ -1,5 +1,54 @@
 {
   "releases": {
+    "1.7.1": [
+      "[New] You can now choose to bring your changes with you to a new branch or stash them on the current branch when switching branches - #6107",
+      "[New] Rebase your current branch onto another branch using a guided flow - #5953",
+      "[New] Recently opened repositories appear at the top of the repository list - #7132",
+      "[New] Suggested next steps now includes suggestion to create a pull request after publishing a branch - #7505",
+      "[Added] .resx syntax highlighting - #7235. Thanks @say25!",
+      "[Added] \"Exit\" menu item now has accelerator and access key - #6507. Thanks @AndreiMaga!",
+      "[Added] Help menu entry to view documentation about keyboard shortcuts  - #7184",
+      "[Added] \"Discard all changes\" action under Branch menu - #7394. Thanks @ahuth!",
+      "[Fixed] \"Esc\" key does not close Repository or Branch list - #7177. Thanks @roottool!",
+      "[Fixed] Warn when renaming a branch with a stash - #7283",
```

Comment:
I think this is captured in the first line item.

> [New] You can now choose to bring your changes with you to a new branch or stash them on the current branch when switching branches - #6107"

Reply:
I left this in because it was a separate dialog to the new flow, but I don't have strong feels about keeping it in...

Reply:
If we do keep it, I'd rather see it marked as **new** since stashing didn't exist in the last release. Calling it a fix wouldn't be accurate.

Reply:
👍 to dropping it from the list